### PR TITLE
Navigation(simple)

### PR DESCRIPTION
**Pull request by TusharJade**
Added Navigation

### CODE REVIEW

**Review by harshitpaliwal95 — COMMENTED** [review]
good to go

File: Components/Navigation/navigation.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            <li class="nav-link"><a class="nav-anchor active"
+                    href="/Components/Navigation/navigation.html">Navigation</a></li>
+        </ul>
+    </nav>
+    <section class="main-container">
+        <h1 class="main-heading">Navigation</h1>
+        <div class="separator"></div>
+        <h3 class="sub-heading">Simple Navigation</h3>
+
+        <div class="header-outerbox">
```
Comment (arulk8) [inline Components/Navigation/navigation.html:39]: use Header tag

File: Components/Navigation/navigation.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            <div class="simple-head">
+                <a href="#" class="header-name">Kharido</a>
+                 <div class="end-headside">
```
Comment (arulk8) [inline Components/Navigation/navigation.html:42]: use Nav tag

**Review by arulk8 — APPROVED** [review]

File: Components/Navigation/navigation.html (hunk 1 above)
Comment (TusharJade) [reply Components/Navigation/navigation.html:39]: Yup will update it

Comment (TusharJade) [reply Components/Navigation/navigation.html:39]: I used header tag twice that why it automatically converted in div IG cause in my vscode from line 40 to 45 I used header tag

Screenshot (30)

File: Components/Navigation/navigation.html (hunk 2 above)
Comment (TusharJade) [reply Components/Navigation/navigation.html:42]: Ok will look into it

File: Components/Navigation/navigation.html (hunk 1 above)
Comment (TusharJade) [reply Components/Navigation/navigation.html:39]: used header tag

**Pull request merged**